pages/cart/cart.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>NeonSports</title>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"rel="stylesheet">
+    <link rel="stylesheet" href="https://musing-varahamihira-c01873.netlify.app/css/main.css">
+    <link rel="stylesheet" href="/pages/productlisting/productlisting.css">
+    <link rel="stylesheet" href="/css/main.css">
+    <link rel="stylesheet" href="/css/common.css">
+    <link rel="stylesheet" href="/pages/cart/cart.css">
+</head>
+<body>
+    <!-- <nav class="nav-container">
+        <div class="nav-left">
+            <h1>NeonSports</h1>
+            <form class="d-flex p-relative">
+                <input type="text" class="searchbar-input" placeholder="Search for products">
+                <span class="material-icons searchbar-icon">search</span>
+            </form>
+            <a href="/pages/auth/login.html" class="btn btn-primary font-size-sm">login</a>
+        </div>
+        
+        <div class="nav-right">
+            
+            <div class="badge-parent">
+                <span class="material-icons"> shopping_cart </span>
+                <div class="badge-with-icon">5</div>
+            </div>
+            
+            <div class="badge-parent">
+                <span class="material-icons"> favorite_border </span>
+                <div class="badge-with-icon">3</div>
+            </div>
+    
+            <div class="badge-parent">
+                <span class="material-icons"> notifications_none </span>
+                <div class="badge-with-icon">1</div>
+            </div>
+        </div>
+    </nav> -->
```

Comment:
Remove commented code

Reply:
thanks for pointing out 👍
but I kept it for a reason